python/tvm/relay/frontend/onnx.py

```diff
@@ -1080,6 +1080,18 @@ class Or(Elemwise):
     def _impl_v7(cls, inputs, attr, params):
         return _op.logical_or(inputs[0], inputs[1])
 
+class Expand(OnnxOpConverter):
+    """ Operator converter for Expand.
+    """
+
+    @classmethod
+    def _impl_v1(cls, inputs, attr, params):
+        assert len(inputs) == 2, "ONNX Expand must have 2 inputs."
+        assert isinstance(inputs[1], tvm.relay.expr.Var), "2nd operand of ONNX Expand must be a constant."
```

Comment:
I don't think we need to assert that it's a constant. We can use `_infer_value_simulated` to get the value if it's not. This is required for some of the BERT models I've been testing.

Reply:
ok, I'll fix it.